lua/go/lint.lua

```diff
@@ -150,10 +150,17 @@ local function do_lint(linter, args)
 end
 
 function M.golangci_lint()
+    if util.binary_exists('golangci-lint') then
+        local status = vim.system({'golangci-lint', 'version', '--short'}):wait()
+        if status.code ~= 0 or string.sub(status.stdout,1,2) ~= "v2" then
+            output.show_warning('GoLint', 'Incompatible golangci-lint version. Try running \'GoUpdateBinaries\'.')
```

Comment:
Shall we `return` here? otherwise, looks good.

Reply:
good catch. fixed!